### PR TITLE
EVG-20777: Run yarn codegen

### PR DESCRIPTION
**Pull request by SupaJoon**
EVG-20777

### Description
I updated test data based on the enums introduced in the codegen

### CODE REVIEW

File: src/pages/distroSettings/tabs/testData.ts [hunk 1]
```diff
@@ -1,20 +1,27 @@
 import {
+  Arch,
```
Comment (minnakt) [inline src/pages/distroSettings/tabs/testData.ts:2]: nice alphabet 😎

File: src/pages/distroSettings/tabs/testData.ts [hunk 2]
```diff
@@ -1,20 +1,27 @@
...
   PlannerVersion,
   Provider,
+  RoundingRule,
 } from "gql/generated/types";
 
 const distroData: DistroQuery["distro"] = {
   __typename: "Distro",
   aliases: ["rhel71-power8", "rhel71-power8-build"],
-  arch: "linux_ppc64le",
+  arch: Arch.LinuxArm_64Bit,
```
Comment (minnakt) [inline src/pages/distroSettings/tabs/testData.ts:20]: I think this should actually be the `Ppc` arch!

File: src/pages/distroSettings/tabs/testData.ts [hunk 3]
```diff
@@ -23,7 +30,7 @@ const distroData: DistroQuery["distro"] = {
     ],
     jasperBinaryDir: "/home/evg/jasper",
     jasperCredentialsPath: "/home/evg/jasper/creds.json",
-    method: "legacy-ssh",
+    method: BootstrapMethod.Ssh,
```
Comment (minnakt) [inline src/pages/distroSettings/tabs/testData.ts:33]: BootstrapMethod.LegacySsh

File: src/pages/distroSettings/tabs/testData.ts [hunk 4]
```diff
@@ -1,20 +1,27 @@
...
 const distroData: DistroQuery["distro"] = {
   __typename: "Distro",
   aliases: ["rhel71-power8", "rhel71-power8-build"],
-  arch: "linux_ppc64le",
+  arch: Arch.LinuxArm_64Bit,
   authorizedKeysFile: "",
   bootstrapSettings: {
     clientDir: "/home/evg/client",
-    communication: "legacy-ssh",
+    communication: CommunicationMethod.Rpc,
```
Comment (minnakt) [inline src/pages/distroSettings/tabs/testData.ts:24]: CommunicationMethod.LegacySsh

File: src/pages/distroSettings/tabs/testData.ts [hunk 5]
```diff
@@ -65,13 +72,13 @@ const distroData: DistroQuery["distro"] = {
...
-    feedbackRule: "",
+    feedbackRule: FeedbackRule.Default,
     futureHostFraction: 0,
-    hostsOverallocatedRule: "",
+    hostsOverallocatedRule: OverallocatedRule.Default,
     maximumHosts: 0,
     minimumHosts: 0,
-    roundingRule: "",
-    version: "utilization",
+    roundingRule: RoundingRule.Up,
```
Comment (minnakt) [inline src/pages/distroSettings/tabs/testData.ts:80]: RoundingRule.Default

Comment (sophstad) [inline src/pages/distroSettings/tabs/testData.ts:80]:
```suggestion
    roundingRule: RoundingRule.Default,
```

**Review by minnakt — APPROVED** [review]

File: src/pages/distroSettings/tabs/testData.ts (hunk 3 above)
Comment (sophstad) [inline src/pages/distroSettings/tabs/testData.ts:33]:
```suggestion
    method: BootstrapMethod.LegacySsh,
```

File: src/pages/distroSettings/tabs/testData.ts (hunk 4 above)
Comment (sophstad) [inline src/pages/distroSettings/tabs/testData.ts:24]:
```suggestion
    communication: CommunicationMethod.LegacySsh,
```

File: src/pages/distroSettings/tabs/testData.ts (hunk 2 above)
Comment (sophstad) [inline src/pages/distroSettings/tabs/testData.ts:20]:
```suggestion
  arch: Arch.LinuxPpc_64Bit,
```

**Review by sophstad — CHANGES_REQUESTED** [review]
Would prefer to use the values closest to those currently present in the test data!

Edit: oops sorry we reviewed at the same time 😅

**Pull request merged**